profile/benchmarking/complex.rb

```diff
@@ -48,15 +48,15 @@ def index_documents(opts = {})
         end
 
         with_cleanup do
-          start = Time.now
+          start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
```

Comment:
Argh I misread it and thought there was no common Module that both Simple and Complex include, but there is! It's Measurable. In that case I'd propose extracting this to something like `current_time`/`monotonic_time` so that this detail doesn't litter the code base - what do you think?

Reply:
Hey @PragTob Thank you for the pull request! I do indeed think extracting this to a common `#curent_time` method would be great. Do you want to make the change or would you rather I merge this, then update it? Thanks again for the opportunity to speak at the meetup last night!

Reply:
I'll gladly do it :) And thanks for the talk :tada: